metc-wastewater-covid-monitor/server.R

```diff
@@ -424,7 +426,8 @@ server <- function(input, output) {
               size = 14,
               family = font_family_list,
               color = councilR::colors$suppBlack
-            )
+            ),
+            rangemode = "nonnegative"
```

Comment:
It appears that "nonnegative" was added to both the "title" structure and also to the "yaxis" structure a few lines below.  I don't think that adding this to the "title" structure will have any effect.  (I could be wrong, though!)